orm/models.py

```diff
@@ -454,6 +467,41 @@ async def update(self, **kwargs) -> None:
 
         await self.database.execute(expr)
 
+    async def bulk_update(
+        self, objs: typing.List[MODEL], fields: typing.List[str]
+    ) -> None:
+        fields = {
+            key: field.validator
+            for key, field in self.model_cls.fields.items()
+            if key in fields
+        }
+        validator = typesystem.Schema(fields=fields)
+        new_objs = [
+            _update_auto_now_fields(validator.validate(value), self.model_cls.fields)
+            for value in [
+                {
+                    key: _convert_value(value)
+                    for key, value in obj.__dict__.items()
+                    if key in fields
+                }
+                for obj in objs
+            ]
+        ]
+        expr = (
+            self.table.update()
+            .where(self.table.c.id == sqlalchemy.bindparam("id"))
```

Comment:
I think you should use Model primary key here. It's not always `id`.

Reply:
Yes you're right.